department-of-veterans-affairs/va.gov-cms
# VACMS-11505: add Last Saved by an Editor field to all  content types

## Description

Closes #11505.

## Testing done
Tested locally. 

## Screenshots
![CleanShot 2022-11-15 at 08 10 04](https://user-images.githubusercontent.com/109987005/201969261-e6ca299a-a04d-4628-b6ad-a071cc65762c.png)
![CleanShot 2022-11-15 at 08 07 53](https://user-images.githubusercontent.com/109987005/201968812-c8b0a3f0-4c09-4d41-b182-4b03753bbf49.png)

## QA steps

Select a piece of content, scroll to the bottom and change the moderation state to Published and click save. In the node view, you can see the field at the bottom of the right sidebar with a date and time. If you go back into the edit form, you will be able to see the information in the top section of the right sidebar. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
